### PR TITLE
Misc. typo and whitespace fixes

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: ElmerGUI/Application/plugins/egmain.cpp [hunk 1]
```diff
@@ -2,7 +2,7 @@
    ElmerGrid - A simple mesh generation and manipulation utility  
    Copyright (C) 1995- , CSC - IT Center for Science Ltd.   
 
-   Author: Peter R�back
+   Author: Peter R�back
```
Comment (luzpaz) [inline ElmerGUI/Application/plugins/egmain.cpp:5]: sorry about this. Using a graphical git client and there must be a bug since it's not showing up in my UI. Will try to revert these changes

Comment (luzpaz) [reply ElmerGUI/Application/plugins/egmain.cpp:5]: darn, I can't revert these changes since it doesn't show up in my UI :angry:  
Not sure how to do this via the CLI

**Comment by raback** [comment]
Thanx!
